packages/fluentui/react-bindings/src/FocusZone/FocusTrapZone.tsx

```diff
@@ -62,6 +63,7 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
     as: 'div',
     isClickableOutsideFocusTrap: true,
     forceFocusInsideTrapOnOutsideFocus: false,
+    innerRef: PropTypes.any,
```

Comment:
This should be under `propTypes` not in `defaultProps` then it should pass

Reply:
Done